acceptance/tests/concurrency/ticket_2659_concurrent_catalog_requests.rb

```diff
@@ -87,7 +87,7 @@
           sleep $sleep_for
           url='https://#{master}:8140/puppet/v3/catalog/#{agent_cert}?environment=production'
           echo "Curling: $url"
-          env PATH='#{agent[:privatebindir]}:${PATH}' curl --tlsv1 -v -# -H 'Accept: text/pson' --cert #{cert_path} --key #{key_path} --cacert #{cacert_path} $url
+          env PATH="#{agent[:privatebindir]}:$PATH" curl --tlsv1 -v -# -H 'Accept: text/pson' --cert #{cert_path} --key #{key_path} --cacert #{cacert_path} $url
```

Comment:
@joshcooper No curl in /opt/puppetlabs/puppet/bin on huaweios.  Also, simply having  /opt/puppetlabs/puppet/bin in the PATH first should be adequate to make sure OUR curl is used (if it's being built).